framework/src/parser/CommandLine.C

```diff
@@ -414,6 +416,10 @@ CommandLine::printUsage() const
 
   Moose::out << "Solver Options:\n"
              << "  See PETSc manual for details" << std::endl;
+
+  // If we get here, we are not running a simulation and should silence petsc's unused options
+  // warning
+  Moose::PetscSupport::setSinglePetscOption("-options_left", "0");
```

Comment:
Changes in this file are unrelated to the new `ignored_options` attribute in `PetscOptions`, but having this here does get rid of some additional unused option warnings.